app/nslc/models.py

```diff
@@ -66,6 +67,7 @@ class Channel(Nslc):
         default=datetime(2599, 12, 31, tzinfo=pytz.UTC))
     network = models.ForeignKey(Network, on_delete=models.CASCADE,
                                 related_name='channels')
+    nslc = models.CharField(max_length=255, null=True)
```

Comment:
This works, although you could also define nslc as a property so it doesn't require an additional column in the db. ArchiveBase has a couple examples in measurements/models.py, serializers.py.

Reply:
I decided to make it a column so we could potentially use it as the pk or indexing